hydro/distribution.yaml

```diff
@@ -3883,6 +3883,17 @@ repositories:
       type: git
       url: https://github.com/UC3MSocialRobots/maggie_devices_msgs.git
       version: hydro-devel
+    release:
+      packages:
+      - eyelids_msgs
+      - ir_controller_msgs
+      - maggie_devices_msgs
+      - motor_controller_msgs
+      - rfid_msgs
```

Comment:
These package names sound very generic. I am not sure if it makes sense to introduce them as-is. May be it would be better to prefix them with `maggie_` in order to not indicate that they are supposed to be generic messages?